tools/ci/install-r-dependencies.sh

```diff
@@ -2,7 +2,21 @@
 set -euo pipefail
 # Context: .github/workflows/*.yaml
 export R_REMOTES_NO_ERRORS_FROM_WARNINGS='true'
+
+# TODO: Remove this once we have either a TileDB-R 0.17.1 depending on core 2.13.1, or, a TileDB-R
+# 0.18 depending on core 2.14.
+#
+# See also https://github.com/single-cell-data/TileDB-SOMA/issues/654
+#
+# * Enable repository from tiledb-inc
+# * Download and install tiledb in R
+Rscript -e '
+    options(repos = c(tiledbinc = "https://tiledb-inc.r-universe.dev", CRAN = "https://cloud.r-project.org"));
+    install.packages("tiledb")
+'
+
```

Comment:
Nit level comment:  I would write/indent it more tightly (two lines max) and possibly also place it below the `remotes` install.  But most importantly it works.